features/machine/autoscaler.feature

```diff
@@ -183,16 +184,15 @@ Feature: Cluster Autoscaler Tests
   Scenario: Machineautoscaler can be deleted when its referenced machineset does not exist
     Given I have an IPI deployment
     And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
 
     Given I store the number of machines in the :num_to_restore clipboard
```

Comment:
I think we should remove all these lines given clipboard is not used anymore.

Reply:
Removed these lines.